frontend-react/src/pages/admin/value-set-editor/ValueSetsIndex.tsx

```diff
@@ -0,0 +1,117 @@
+import { Helmet } from "react-helmet";
+import React, { useState } from "react";
+
+import Table, {
+    ColumnConfig,
+    DatasetAction,
+    LegendItem,
+    TableConfig,
+} from "../../../components/Table/Table";
+
+/* FAUX DATA AND STUFF TO BE REMOVED WHEN IMPLEMENTING THE API */
+interface ValueSet {
+    value: string;
+    header: string;
+    type: string;
+}
+
+const Legend = ({ items }: { items: LegendItem[] }) => {
+    const makeItem = (label: string, value: string) => (
+        <div key={label} className="display-flex">
+            <b>{`${label}:`}</b>
```

Comment:
I had this in the Table file at first but I think it makes more sense to leave this to a page file. Not all legends will have the same needs, so we can make these on a per-use basis.